src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Network/ApplicationGateway/Add-ArmApplicationGatewayUrlPathMap.ps1

```diff
@@ -1,4 +1,4 @@
-function Add-ArmApplicationGatewayRequestRoutingRule {
+function Add-ArmApplicationGatewayUrlPathMap {
```

Comment:
I don't really like how we structure that function. Let's discuss together

Reply:
added new function to add a path to a map, let me know if anything can be improved